client/src/App.jsx

```diff
@@ -11,7 +11,7 @@ const App = () => {
     <ProductDetails
       productId={40344} //TODO: create state to control productId
     />
-    <RelatedItems />
+   <RelatedItems/>
```

Comment:
For the initial version of the app you should be passing in the productId as a prop here and using that parameter in the rest of your components. 
As is, it looks like you're hardcoding a productId (37311) on numerous lines in your components.